src/DressMeUpApi/app/models/user.model.js

```diff
@@ -0,0 +1,122 @@
+import Clothes from "./clothes.model";
+import Outfit from "./outfit.model";
+
+module.exports = (sequelize, Sequelize) => {
+
+    const User = sequelize.define("User", {
+
+        email: {
+            type: Sequelize.STRING,
+            allowNull: true,
+            validate: {
+                customValidator(value) {
+                  if (value === null && this.phoneNumber === null) {
+                    throw new Error("Email must be provided");
+                  }
+                },
+            },
+            unique: true
+        },
+        
+        phoneNumber: {
+            type: Sequelize.STRING,
+            allowNull: true,
+            validate: {
+                customValidator(value) {
+                  if (value === null && this.email === null) {
+                    throw new Error("Phone number must be provided");
+                  }
+                },
+            },
+            unique: true
+        },
+        password: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        fullName: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        username: {
+            type: Sequelize.STRING,
+            allowNull: false,
+            unique: true
+        },
+        admin: {
+            type: Sequelize.BOOLEAN,
+            allowNull: false,
+            defaultValue: false
+        },
+        
+        createdAt: {
+            type: Sequelize.DATE,
+            allowNull: false,
+            defaultValue: Sequelize.NOW
+        },
+        updatedAt: {
+            type: Sequelize.DATE,
+            allowNull: false,
+            defaultValue: Sequelize.NOW
+        },
+
+        clothes: {
+            type: Sequelize.ARRAY(Sequelize.INTEGER),
```

Comment:
Did you try to use the ARRAY properties when the app is running and connected to the database? Per sequelize documentation, it only works with PostgreSQL and it didn't work, when I tried it with colors and seasons etc.